actions/webhook.go

```diff
@@ -32,7 +32,10 @@ type WebhookEventInfo struct {
 	Metadata      map[string]string `json:"metadata,omitempty"`
 }
 
-const webhookClientTimeout = 5 * time.Minute
+const (
+	webhookClientTimeout      = 1 * time.Minute
+	webhookTimeoutPropertyKey = "timeout"
```

Comment:
So the user can supply the value for this?

Reply:
Maybe it should start with some prefix to differentiate between lakeFS properties and custom user properties.

Reply:
All the properties are associated with the hook implementation. if you want custom properties per webhook it should be property under properties.